.env.example

```diff
@@ -93,14 +59,47 @@ PW_STATPING=5ecr3t
 
 STATPING_TOKEN=
 
-PHP_APP_KEY=YdEgGZtx98WIBOKJky3gBvcnE+1ZW9+40HHSI0p+vWQ=
-RUBY_APP_KEY=YdEgGZtx98WIBOKJky3gBvcnE+1ZW9+40HHSI0p+vWQ=
+SSH_PORT=
+ELK_VERSION=
 
+#possibly remove?
```

Comment:
I think we can safely remove it (and wiki at all) and moving wiki plugin to other repo, as it's not the part of oms-docker and is hosted on zeus

Reply:
yes but the objective imho is to bring it to us, we also can add automatic backups etc

Reply:
hm but ok i will remove it so it is in the same commit, we can look back here for the code